stylus/components/button.styl

```diff
@@ -76,99 +76,43 @@ $button
 $button--regular  // Deprecated
     @extend $button
 
-$button--secondary
-    background-color  white
-    color             black
-    border-color      silver
-
-    svg
-        fill currentColor
-
-    &[disabled]
-    &[aria-disabled=true]
+highlight = { name: 'highlight', bgColor: emerald, bdColor: emerald, focusColor: malachite, txtColor: white}
+danger = { name: 'danger', bgColor: pomegranate, bdColor: pomegranate, focusColor: monza, txtColor: white}
+alpha = { name: 'alpha', bgColor: transparent, bdColor: white, focusColor: scienceBlue, txtColor: white}
+secondary = { name: 'secondary', bgColor: white, bdColor: silver, focusColor: silver, txtColor: black}
+danger-outline = { name: 'danger-outline', bgColor: white, bdColor: yourPink, focusColor: yourPink, txtColor: pomegranate}
```

Comment:
As we aim to be able to easy build ButtonText  (or _no-outline_ buttons) from themes, and knowing that in those buttons color are "inverted" what do you think about naming attributes something like:
```styl
highlight  = { name: 'highlight', mainColor: emerald, secondaryColor: emerald, activeColor: malachite, contrastColor: white }
```